benchmarks/kftv2-mpi-ddp-sft-ddp/1.5b/train_sft_ddp.py

```
@@ -0,0 +1,290 @@
+        post_warmup = step_times[WARMUP_STEPS:] or step_times
+        avg_step = sum(post_warmup) / len(post_warmup)
+        min_step = min(post_warmup)
+        max_step = max(post_warmup)
+        throughput = BATCH_SIZE * world_size / avg_step
+        peak_mem = torch.cuda.max_memory_allocated() / 1e9
+        avg_loss = sum(losses[-len(post_warmup):]) / len(post_warmup)
+
+        print(f"\n{'=' * 60}", flush=True)
+        print("BENCHMARK RESULTS (DDP + MPI)", flush=True)
+        print(f"{'=' * 60}", flush=True)
+        print(f"  Backend:            MPI (via DDP)", flush=True)
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)